core/src/main/java/org/radargun/stages/test/BaseTestStage.java

```diff
@@ -28,6 +28,14 @@
    @Property(converter = TimeConverter.class, doc = "Benchmark duration. You have to set either this or 'totalNumOperations'.")
    public long duration = 0;
 
+   /**
+    * The stage have a duration, for example: 8 minutes. When doing a scalability test I would like to start the node
+    * later to simulate a scale up test. Decrease duration could be something like: #{${slave.index}*2}m
+    * In this case, the second slave will start two minutes later
+    */
+   @Property(converter = TimeConverter.class, doc = "You have the possibility to decrease the stage duration in runtime. Default zero")
```

Comment:
I'm still trying to understand what this does from the doc string, and I don't get it. Is this to get an accurate time that this node was running the test for the stats? It seems like there are too many config options that have to be set. (i.e. `stagger-slave-startup-mode`, `delay-between-starting-slaves`, `decrease-duration`, and `duration`)

Reply:
I think have a new `Stage` will make things easier. Let me work on that